scripts/coverage.sh

```diff
@@ -22,4 +20,7 @@ if [ ! $trpc_running ]; then
   trpc_pid=$!
 fi
 SOLIDITY_COVERAGE=true && ./node_modules/.bin/solidity-coverage
```

Comment:
Not introduced in this PR, but this line defines a variable that is not exported. Was the intention to define an env var for the `solidity-coverage` process? If so, `&&` should be removed.